styles/nextjs/catppuccin.user.css

```diff
@@ -0,0 +1,247 @@
+    @base: @catppuccin[@@lookup][@base];
+    @mantle: @catppuccin[@@lookup][@mantle];
+    @crust: @catppuccin[@@lookup][@crust];
+    @accent-color: @catppuccin[@@lookup][@@accent];
+
+    --geist-foreground: @text;
+    --geist-background: @mantle;
+    --accents-1: @base;
+    --accents-2: @surface0;
+    --accents-3: @surface1;
+    --accents-4: @surface2;
+    --accents-5: @overlay0;
+    --accents-6: @overlay1;
+    --accents-7: @overlay2;
+    --accents-8: @subtext0;
+    --geist-link-color: @rosewater;
+    --geist-selection: @subtext0;
+    --geist-success: @accent-color;
+    --geist-error: @red;
+    --geist-cyan: @teal;
+
+    --ds-background-100: @base;
+    --ds-background-200: @surface0;
+    --ds-gray-100: @base;
+    --ds-gray-200: @surface0;
+    --ds-gray-300: @surface1;
+    --ds-gray-400: @surface0;
+    --ds-gray-700: @subtext0;
+    --ds-gray-1000: @text;
+    --ds-gray-900: @subtext0;
+    --ds-gray-800: @subtext1;
+
+    --ds-red-800: @red;
+    --ds-red-900: lighten(@red, 5%);
+
+    --ds-blue-300: @surface0;
+    --ds-blue-900: @accent-color;
+
+    /* Links vertical line */
+    .bg-blue-600 {
+      background-color: @accent-color;
+    }
+
+    /* App Router icon */
+    --ds-blue-700: @blue;
+
+    /* Pages Router icon */
+    --ds-purple-700: @mauve;
+
+    --ds-blue-400: {
+      color: @base;
+    };
+
+    --ds-purple-400: {
+      color: @base;
+    };
+
+    --tw-gradient-top: transparent;
+
+    a h3 {
+      color: @accent-color;
+    }
+
+    p a {
+      color: @accent-color;
+    }
+
+    ul li::marker {
+      color: @accent-color;
+    }
+
+    .text-blue-700 {
+      color: @accent-color;
+    }
+
+    svg {
+      --geist-stroke: var(--geist-background) !important;
+    }
+
+    button.navbar_search__dZT2b {
+      background-color: @mantle !important;
+    }
+
+    input::placeholder,
+    button[class*="navbar_search"] {
+      color: @text;
+    }
+
+    code {
+      color: @accent-color !important;
+    }
+
+    .shadow,
+    .shadow-md,
+    .shadow-sm {
+      box-shadow: none !important;
+    }
+
+    .code-block_wrapper__t6FCO {
+      border-color: 1px solid @surface0 !important;
+    }
+
+    .group:hover .dark\:group-hover\:bg-white {
+      background-color: @subtext1;
+    }
+
+    .group:hover .dark\:group-hover\:text-black {
+      color: @crust;
+    }
+
+    [data-docs-container] code:not(pre code),
+    [data-docs-table-of-contents] code:not(pre code) {
+      border-color: 1px solid @surface0 !important;
+      background-color: @mantle !important;
+    }
+
+    td a {
+      color: @accent-color;
+    }
+
+    .animated-optimizations_window__CjM85 {
+      background: @mantle !important;
+    }
+
+    .features_card__xYla9,
+    .foundation_root__wm0ez,
+    .foundation_card__v7VKB {
+      background: @base;
+    }
+
+    .foundation_cpu__ciXpm {
+      background: @mantle;
+    }
+
+    .header_header__zJOD0 {
+      background: @base;
+    }
```

Comment:
The following suggestion themes the mobile/small screen nav menu:
![Screenshot 2024-02-08 at 15 44 08 (Arc)](https://github.com/catppuccin/userstyles/assets/47499684/8785e4c3-dc05-4511-864f-0a5478b17553)

```suggestion
    .header_header__zJOD0 {
      background: @base;
    }
    .mute, .subtitle {
      color: @subtext0;
    }

    .mobile-menu_mobileMenu__xqbOP a.mobile-menu_selected__G55UL {
      color: var(--geist-background);
    }

    .dark-theme .dark\:bg-black\/50 {
      background-color: var(--geist-background);
    }
```